web/static/web/css/header.css

```diff
@@ -1,144 +1,274 @@
+#side-nav > .item:hover .bubble-background {
+    width: calc(100% + var(--bubble-background-extra-width));
+    height: calc(var(--font-size) + 2em);
+    opacity: 1;
 }
 
-.lang-form {
-    display: none;
+#nav > .item > .text,
+#side-nav > .item > .text {
+    z-index: 1;
+    transition: color var(--dur) ease;
 }
 
-@media screen and (max-width: 900px) {
-    .limg.large {
-        display: none;
-    }
+#nav > .item .make_bg_yellow.bubble-background {
+    filter: brightness(90%);
+}
 
-    .limg.small {
-        display: none;
-    }
+#nav > .item .make_bg_turquoise.bubble-background {
+    filter: brightness(90%);
+}
 
-    .limg.medium {
-        display: inline;
+#nav > .item .make_bg_blue.bubble-background,
+#side-nav > .item .make_bg_blue.bubble-background {
+    filter: brightness(130%);
+}
+
+#side-nav {
+    --estimated-max-name-length: 10em;
+
+    margin: 0;
+    justify-self: end;
+    display: grid;
+    grid-template-columns: var(--estimated-max-name-length) calc(var(--logo-padding) + 10px);
+    justify-items: end;
+    font-size: var(--font-size);
+}
+
+#side-nav > .item {
+    margin-right: 0;
+}
+
+#nav-user-dropdown .menu {
+    background-color: rgb(34, 43, 52);
+    border-radius: 7px;
+    /* For some reason doesn't work with `border: 2px rgb(220, 221, 222)` */
+    border: 2px solid rgb(220, 221, 222);
+}
+
+#nav-user-dropdown .menu > .item {
+    /* Fixes weird centering of dropdown items, caused by Semantic UI setting dropdown menu to `display: flex !important`
+       when it's too close to the right window edge */
+    align-self: auto;
+}
+
+#nav-user-dropdown .menu > .item:hover {
+    background-color: rgba(255, 255, 255, .08) !important; /* Overrides Semantic UI */
+}
+
+#side-nav .lang-link {
+    position: absolute;
+    margin: 0;
+    top: 0;
+    right: 0;
+}
+
+#side-nav .lang-link .link {
+    margin: 0;
+    align-self: start;
+}
+
+#side-nav .lang-link .link .text {
+    margin-right: 0.5em;
+}
+
+#side-nav .lang-link .link .globe.icon {
+    margin: 0;
+}
+
+/* Narrower space between elements in navbar */
+@media screen and (max-width: 1100px) {
+    #header {
+        --logo-padding: 1em;
+        --nav-space-between: 1em;
     }
 }
 
-@media screen and (max-width: 730px) {
-    .limg.large {
-        display: inline;
+/* Switch to mobile layout with burger */
+@media screen and (max-width: 992px) {
+    #header {
+        --burger-space: minmax(calc(4em + var(--burger-margin)), 1fr);
+
+        grid-template-columns: var(--burger-space) auto var(--burger-space);
+        grid-template-rows: var(--header-height);
     }
 
-    .limg.small {
-        display: none;
+    #header.active {
+        height: auto;
     }
 
-    .limg.medium {
-        display: none;
+    #header div.logo-centering {
+        display: inline;
     }
 
-    .nav {
-        grid-template-columns: 1fr 3fr;
+    #header .logo {
+        grid-template-columns: 1em var(--logo-space) 1em;
     }
```

Comment:
You could just remove this. For screen sizes larger than 992px, `grid-template-columns` is set to `var(--logo-padding) var(--logo-space) minmax(var(--nav-space-between), var(--logo-padding))` which due to the redefinition of `--logo-padding` to `1em` for screen sizes smaller than or equal to `1100px` (line 165 - 170), is equivalent to the definition here.

Reply:
I'd rather keep it - or optimally change it to `var(--logo-padding) var(--logo-space) var(--logo-padding)` for verbosity, to not make it dependent of `var(--nav-space-between)`; because if the latter variable were to increase, the logo would get offset from the center.

Reply:
I see the point of not being dependent on `var(--nav-space-between)`. I feel that the explicit declaration you mention, `var(--logo-padding) var(--logo-space) var(--logo-padding)`, would be better for readability.